lib/ssj/workflow/import.rb

```diff
@@ -46,13 +49,18 @@ def import_process_library
             process_weight = row[19]&.strip # convert to integer
             process_effort = {"S": 0, "M": 10, "L": 100}[process_weight]
             process_tag = row[7]&.strip # tag process.
-            process_obj = ::Workflow::Definition::Process.create! version: default_version, title: process_title, description: process_description, effort: process_effort
+            process_position += ::Workflow::Definition::Process::DEFAULT_INCREMENT
```

Comment:
FYI, you can also create an after hook to set the process position on save. This is what we're doing for instance steps. Would prevent us from creating objects without a valid position.